jujugui/static/gui/src/app/components/store/store.js

```diff
@@ -153,7 +155,7 @@ class Store extends React.Component {
           </div>
         </div>
         <div>
-          <img src={this._generateLocalImagePath('write-your-own.png')} />
+          <img src={images['write-your-own.png']} />
```

Comment:
I'm curious what this gets generated as? Is it just the path on disk relative for the parcel root?